### DISCUSSION TITLE
How can I filter out lines with the first latter as uppercase?

### DISCUSSION THREAD

**Discussion by otakutyrant** [nushell/nushell#10026] [Q&A] [answered]
I tried `open learned_words.txt | lines | where { |$it| $it | str contains '[ABCDEFGHIJKLMNOPQRSTUVWXYZ]'}`, but it didn't work.

**Comment by panicbit** [comment, marked as answer]
I would do something like `open learned_words.txt | lines | where $it =~ '^[A-Z]'`

**Reply by otakutyrant** [reply]
My appreciation.
